recipes/arrow-adbc/meta.yaml

```diff
@@ -0,0 +1,203 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: arrow-adbc-split
+  version: {{ version }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/apache-arrow-adbc-{{ version }}/apache-arrow-adbc-{{ version }}.tar.gz
+  sha256: a50f3dac985883042152938bdd69c843708fd50508e0f15ba3a179619c11f7b9
+  patches:
+    # Backport https://github.com/apache/arrow-adbc/issues/304, drop in 0.2.0
+    - patches/0001-fix-c-driver-postgresql-define-ntohll-etc-for-macOS.patch
+
+build:
+  number: 0
+
+outputs:
+  - name: adbc-driver-manager-cpp
```

Comment:
Outputs with headers and shared libraries should have run_exports which pin an ABI compatible version of the output.